### PR TITLE
prep-css-flexbox - Matthew Cha

### PR DESCRIPTION
**Pull request by matt-cha**
display-flex
align-items
flex-wrap
justify-content

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Matthew, please attach the rest of your screenshots to the Pull Request

**Comment by matt-cha** [comment]
Hi Tim, sorry about that! I think I may need to add them each as an individual comment since adding all 4 at the same time only uploads one

**Comment by matt-cha** [comment]
flex-wrap

**Comment by matt-cha** [comment]
justify-content

**Comment by matt-cha** [comment]
align-items

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Matthew! since `flex-basis` is a "flex item" property, you'll want to move all of your `flex-basis` properties to a rule set for `.card-wrapper`. 

Also, when doing fractions in CSS, we always use `calc()` like this, for example:

```css
.card-wrapper {
  flex-basis: calc(100% / 3);
}
```

**Comment by matt-cha** [comment]
Screen Shot 2022-09-27 at 9 40 16 AM

Uploaded new changes for the .card-wrapper!

File: prep-css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -9,11 +9,11 @@
     #card-deck {
       display: flex;
       flex-wrap: wrap;
-      flex-basis: 1/6;
       padding: 1rem;
 }
     .card-wrapper: {
       padding: 1rem;
+      flex-basis : calc(100% / 6)
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html:16]: Please check the rest of your code for this fix as well.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-flexbox/flex-wrap.html (hunk 1 above)
Comment (thebearingedge) [reply prep-css-flexbox/flex-wrap.html:16]: https://github.com/matt-cha/prep-0922-code-solutions/pull/8/files

**Comment by matt-cha** [comment]
I believe I have fixed the remaining code thank you for being thorough Tim!

**Comment by matt-cha** [comment]
Screen Shot 2022-09-27 at 5 18 19 PM

**Comment by matt-cha** [comment]
Screen Shot 2022-09-27 at 5 19 13 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**